### DISCUSSION TITLE
running playbook from folder in repo

### DISCUSSION THREAD

**Discussion by thedom4302** [semaphoreui/semaphore#1602] [Q&A] [answered]
Is there any way to run a playbook from folder in my git repository rather than have playbooks directly nested under the root?

**Comment by SamB-GB** [comment]
I cam here looking for exactly this question, I am using Git with SSH.

**Comment by Mellowlynx** [comment, marked as answer]
folder/name.yml should work, It's what I use.

**Comment by thedom4302** [comment]
Oh my days…. Thanks thought I would of tried that obviously not.

Cheers,
Dom
